src/main/scala/OiOsmDiff.scala

```diff
@@ -3,54 +3,80 @@ package oiosmdiff
 import java.net.URI
 
 import com.typesafe.scalalogging.LazyLogging
-import geotrellis.proj4.WebMercator
+
+import geotrellis.proj4.{WebMercator, LatLng}
+import geotrellis.spark.SpatialKey
 import geotrellis.spark.tiling.ZoomedLayoutScheme
-import geotrellis.vector.{MultiLine, MultiLineFeature}
-import geotrellis.vectortile.VString
+import geotrellis.vector.{Geometry, Feature}
+import geotrellis.vectortile.{VString, VectorTile}
+
 import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.functions._
 import org.apache.spark.rdd.RDD
+
+import org.locationtech.jts.geom.{Geometry => JTSGeometry}
 import org.locationtech.geomesa.spark.jts._
+
 import vectorpipe._
 import vectorpipe.GenerateVT.VTF
 import vectorpipe.functions.osm._
 
-class OiOsmDiff(osmOrcUri: URI, oiGeoJsonUri: URI)(@transient implicit val spark: SparkSession)
-    extends LazyLogging
-    with Serializable {
 
+class OiOsmDiff(
+  osmOrcUri: URI,
+  oiGeoJsonUri: URI
+)(@transient implicit val ss: SparkSession) extends LazyLogging with Serializable {
   def saveTilesForZoom(zoom: Int, outputS3Prefix: URI): Unit = {
-
-    spark.withJTS
-
     val badRoads = Seq("proposed", "construction", "elevator")
 
-    val osmData = OSM.toGeometry(spark.read.orc(osmOrcUri.toString))
+    val osmData = OSM.toGeometry(ss.read.orc(osmOrcUri.toString))
+
     val osmRoadData = osmData
+    // does isRoad check to see if the geometry is valid?
```

Comment:
It [doesn't look like it](https://github.com/geotrellis/vectorpipe/blob/v1.0.0-RC1/src/main/scala/vectorpipe/functions/osm/package.scala#L249-L250) so I'll add the isvalid check separately.